src/ClusterManager/job_manager.py

```diff
@@ -618,7 +619,7 @@ def AutoApproveJob(job):
 
 UnusualJobs = {}
 
-def UpdateJobStatus(job):
+def UpdateJobStatus(job, notifier):
```

Comment:
should we add the default value to notifier for backwards compatibility. 
def UpdateJobStatus(job, notifier = None):
......
if notifier is not None:
   XXX